format/Flight.proto

```diff
@@ -301,6 +301,36 @@ message Location {
  */
 message Ticket {
   bytes ticket = 1;
+  // Data representing some part of the data retrievable by the ticket.
+  //
+  // `inlined_completeness` indicates what part of the data retriavable
+  // by the ticket this represents. This is provided as an optimization for
+  // client/server applications that want to reduce latency to first result
```

Comment:
Does "reduce latency to first result" sound like the inlined data is always for first rows (`.head()`)?

Reply:
It could, open to suggestions on phrasing.

Reply:
We may mention it as "to get partial result" or "to get preview result".

Reply:
Thanks, tried rephrasing a little bit.